monai/utils/feature_flag.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import annotations
+
+import os
+from contextlib import contextmanager
+
+FEATURE_FLAG_PREFIX = "MONAI_FEATURE_ENABLED_"
+
+
+class FeatureFlag:
```

Comment:
This class could use docstring to describe what it's doing and what it's for. There's a number of other places that we use environment variables we could potentially adapt later to use this.